packages/service-provider-core/src/result.ts

```diff
@@ -1,4 +1,4 @@
 /* eslint no-undef: 0 */
-type Result = any;
+type Result = unknown;
```

Comment:
This may be a bit annoying but it’s probably also going to make us better at actually describing the types that we work with :wink: